app/controllers/podcast_ownerships_controller.rb

```diff
@@ -0,0 +1,42 @@
+class PodcastOwnershipsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new
+    @podcast_ownership = PodcastOwnership.new
+  end
+
+  def create
+    @podcast_ownership = PodcastOwnership.new(podcast_ownership_params)
+
+    if @podcast_ownership.save
+      flash[:success] = "Owner created successfully"
+      redirect_to pod_path
+    else
+      flash[:error] = "User is already an owner"
+      format.html { render :new }
+    end
+  end
+
+  def edit
+    @podcast_ownership = PodcastOwnership.find(params[:id])
+  end
+
+  def update
+    @podcast_ownership = PodcastOwnership.find(params[:id])
+
+    if @podcast_ownership.update(podcast_ownership_params)
+      @podcast_ownership.save
+      flash[:success] = "Updated successfully"
+      redirect_to pod_path
+    else
+      flash[:error] = "Error updating!"
```

Comment:
It's probably better to add `@podcast_ownership.errors_as_sentence` to the error message, this way you can tell the client which errors are present:

https://github.com/forem/forem/blob/910f79e04e402214f33600cddb44367e9c6e5d45/app/models/application_record.rb#L56-L58